packages/ui-components/src/components/Map/MarkerLayer.js

```diff
@@ -19,13 +19,9 @@ const ShadedPolygon = styled(Polygon)`
   }
 `;
 
-const getText = (measure, serieses) => {
-  const { name } = measure;
+const getText = ({ name, ...measure }, serieses) => {
```

Comment:
remove the name key from the measure data as it's not expected in `getSingleFormattedValue`